manual.org

```diff
@@ -0,0 +1,233 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#+title: Emacs package for Bazel
+#+language: en
+#+options: author:nil date:nil
+#+export_file_name: bazel.el.texi
+#+texinfo_filename: bazel.el.info
+#+texinfo_dir_category: Emacs
+#+texinfo_dir_title: bazel.el
+#+texinfo_dir_desc: Working with the Bazel build system
```

Comment:
GitHub preview has some issues: it has lines that contain only `#`.
Is it a problem with the file or with the preview?

Reply:
I think the latter, at least the Texinfo output doesn't have those artifaces.